deltacat/compute/stats/utils/intervals.py

```diff
@@ -1,62 +1,88 @@
-from typing import Set, Tuple
-from collections import Iterable
+from typing import Set, Tuple, List, Iterable, Union
 
+UnboundedRange = Union[int, None]
+NumericalRange = Union[int, float]
 
```

Comment:
```suggestion
DeltaRange = Union[UnboundedRange, NumericalRange]
```

Reply:
Also, since all of these are really just one side of a range, it may be more clear to rename `UnboundedRange` to `DeltaPosition`, `NumericalRange` to `NumericDeltaPosition`, and then define `DeltaRange = Tuple[DeltaPosition, DeltaPosition]` for use in the top-level, user-facing APIs.